pypresence/utils.py

```diff
@@ -1,5 +1,5 @@
-"""Util functions that are needed but messy."""
-import asyncio
+"""Util functions that are messy but needed."""
```

Comment:
Look, I'll level with you, this is a development comment. It's not here for anyone really but the maintainers, and I really don't see the need in correcting grammar especially in comments lol

Reply:
The PR isn't solely a grammar-based PR. You don't need anything tbh, I'm just trying to improve the code base in general.